jobs-builder/jobs/cc.yaml

```diff
@@ -141,3 +216,15 @@
       - CC_SKOPEO_CRI_CONTAINERD_K8S
     jobs:
       - '{repo}-CCv0-{os}-{arch}-{ci_job}-PR'
+- project:
+    name: "Generate jobs for the Confidential Container Operator"
+    os:
+      - ubuntu-20.04
+    arch:
+      - x86_64
+    container_runtime:
+      - containerd
+    runtimeclass:
+      - kata-qemu
```

Comment:
Nice, this is the way we should specify which runtime class to use, right?

Reply:
Exactly. I'm working on changes to allow the CI scripts to get the runtimeclass as parameter; so we will have jobs for kata-qemu, kata-clh, kata-qemu-tdx (this one enabled behind the firewall) and so soon.